src/parser/classish_statement.rs

```diff
@@ -34,16 +35,15 @@ impl Parser {
             return self.method(ClassishDefinitionType::Interface, vec![]);
         }
 
-        let member_flags = self.class_members_flags()?;
+        let member_flags = self.interface_members_flags()?;
 
-        match &self.current.kind {
+        peek_token!([
```

Comment:
difference between `peek_token` and `expect_token` is that `peek_token` doesn't call `self.next();`.